apps/dashboard/app/javascript/dynamic_forms.js

```diff
@@ -110,6 +111,7 @@ function memorizeElements(elements) {
   elements.each((_i, ele) => {
     formTokens.push(mountainCaseWords(shortId(ele['id'])));
     optionForHandlerCache[ele['id']] = [];
+    exclusiveOptionForHandlerCache[ele['id']] = [];
```

Comment:
Is this a bug? I'm not 100% why we're populating these handler cache's here. Looks to be a part of the very initial commit, so I can't tell why we're doing this.

Reply:
I'm don't why it's here either - the test fails without it, though.

Reply:
Seems like we should follow this pattern. Can you move these two lines in the function so that we initialize it just before we check it?


https://github.com/OSC/ondemand/blob/a9569326c1e51967f47b670f3163b307a5fab437/apps/dashboard/app/javascript/dynamic_forms.js#L183-L185